config/routes.rb

```diff
@@ -8,9 +8,12 @@
 
   resources :trails, only: %i[index show] do
     resources :trips, only: :create
+  end
     # collection do
     #   get :mine
     # end
+
+  resources :trips, only: %i[index show] do
+    resources :reviews, only: %i[new create show index]
```

Comment:
don't think we need an index/new/show page for reviews, since reviews and the new review form will only be shown on the trail/trip show page

Reply:
removed